packages/e2e-test-utils-playwright/src/request-utils/users.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Internal dependencies
+ */
+import type { RequestUtils } from './index';
+
+/**
+ * Create user using REST API.
+ *
+ * @param {} this RequestUtils.
+ * @param {string} username User Name.
+ * @param {string} firstName First Name.
+ * @param {string} lastName Last Name.
+ */
+async function createUser(
+	this: RequestUtils,
+	username: string,
+	firstName: string,
+	lastName: string
+) {
+	// Create User
+	// https://developer.wordpress.org/rest-api/reference/users/#create-a-user
+	await this.rest( {
+		method: 'POST',
+		path: '/wp/v2/users',
+		params: {
+			username,
+			first_name: firstName,
+			last_name: lastName,
+			email: username + '@example.com',
+			password: 'secret',
+		},
+	} );
+}
```

Comment:
It'd be good to make this accept an object:
```js
await requestUtils.createUser( {
    username: 'testuser',
    firstName: 'Jane',
    lastName: 'Doe'
);
```

I also implemented this in another PR that's still open, so feel free to use that as a reference -
https://github.com/WordPress/gutenberg/pull/42695/files#diff-c364aa42f972f12234991ab634238513f51e6438661ba139e339df2f305638e5R42-R48

Reply:
#42657 is now merged so you should be able to update this branch to use that version of `createUser`.